### ISSUE TITLE
[Regression] Print crash

### ISSUE THREAD

**Issue by MurakamiShinyu** [vivliostyle/vivliostyle.js#1095]
**Describe the bug**

The following simple HTML+CSS crashes printing from Vivliostyle Viewer (or CLI).

```html
<style>
  @page {
    size: a5;
  }
  h1 {
    margin-top: 40mm;
  }
</style>
<h1>Hello</h1>
```

**To Reproduce**
1. Go to https://vivliostyle.org/viewer/#src=data:,%3Ch1%3EHello%3C/h1%3E&style=data:,/*%3Cviewer%3E*/%0A@page%20%7B%20size:%20a5;%20%7D%0A/*%3C/viewer%3E*/%0Ah1%20%7B%0A%20%20margin-top:%2040mm;%0A%7D
2. Click the Print button
3. waiting the print preview ...

**Screenshots**
![Screenshot 2023-01-19 at 0 27 49](https://user-images.githubusercontent.com/3324737/213217877-957da77f-04e1-4df3-a411-82f2904a0336.png)

This problem does not happen with the previous version v2.20.0:
https://vivliostyle.github.io/viewer/v2.20.0/#src=data:,%3Ch1%3EHello%3C/h1%3E&style=data:,/*%3Cviewer%3E*/%0A@page%20%7B%20size:%20a5;%20%7D%0A/*%3C/viewer%3E*/%0Ah1%20%7B%0A%20%20margin-top:%2040mm;%0A%7D

**Comment by hkwi** [comment]
Firefox/Windows 11 looks OK.

**Comment by MurakamiShinyu** [comment]
This seems to be a side effect of
- https://github.com/vivliostyle/vivliostyle.js/pull/1079

that is effective only with Chromium (Chrome).

The problem does not happen if smaller `pixelRatio` (default: 8) is specified:
https://vivliostyle.org/viewer/#src=data:,%3Ch1%3EHello%3C/h1%3E&style=data:,/*%3Cviewer%3E*/%0A@page%20%7B%20size:%20a5;%20%7D%0A/*%3C/viewer%3E*/%0Ah1%20%7B%0A%20%20margin-top:%2040mm;%0A%7D&pixelRatio=4

**Comment by hkwi** [comment]
```
<style>
  h1 {
    margin-top: 90px;
  }
</style>
<h1>Hello</h1>
```
Chrome 109.0.5414.75 / Windows 11 でサイズを変えて試してみました。

- default pixelRatio
  - 89px OK
  - 90px NG
  - 23mm OK
  - 24mm NG
  - 67pt OK
  - 68pt NG
- pixelRatio = 4
  - 179px OK
  - 180px NG
- pixelRatio = 2
  - 359px OK
  - 360px NG

**Comment by MurakamiShinyu** [comment]
Chrome Beta (110.0.5481.38) と Canary (111.0.5546.0) でテストしたところ、この問題は再現しませんでした。直っているよう。

**Comment by MurakamiShinyu** [comment]
Chromium bug (fixed):

Issue 1400918: Print preview hanging in v108 for certain HTML pages
https://bugs.chromium.org/p/chromium/issues/detail?id=1400918

> Verified the fix on latest chrome canary #110.0.5532.0 using Windows 10, Linux Debian and Mac 13.0
> Observed that the print preview is rendering correctly.

**Comment by MurakamiShinyu** [comment]
Chrome 110 Stable Release is scheduled for 2023-02-07.
https://chromiumdash.appspot.com/schedule

Vivliostyle CLI uses Chromium via [Playwright](https://github.com/microsoft/playwright).
The Chromium version used by Playwright almost corresponds to the beta release of Chome, so the next Playwright version v1.30.0 which uses Chromium 110 will be released soon.

**Comment by MurakamiShinyu** [comment]
This problem is resolved in Vivliostyle CLI v6.2.0 which uses Chromium 110.

However this problem still occurs with the Chrome current version (109) with Vivliostyle Viewer. I will close this issue when Chrome 110 is released.

**Comment by MurakamiShinyu** [comment]
Chrome 110 has been released, and I confirmed that this problem was resolved.
